catalogue/webapp/.env

```diff
@@ -0,0 +1 @@
+PORT=3001
```

Comment:
should this be in here?

Reply:
This file shouldn't be committed - removed it.

Reply:
I think it's still here